.github/workflows/botonic-plugin-contentful-tests.yml

```diff
@@ -4,10 +4,9 @@ on:
   push:
     paths:
       - '*'
```

Comment:
sorry, can you add '*' and 'packages/*' for all packages?

Reply:
included in https://github.com/hubtype/botonic/pull/1177/commits/7e317273c5bb55d5d76562a61db9d4b21bf14d58